Week1/homework/app.js

```diff
@@ -2,10 +2,136 @@
 
 {
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'the_road',
+    'one_hundred_years_of_solitude',
+    'where_late_the_sweet_birds_sang',
+    'to_kill_a_mockingbird',
+    'the_help',
+    'never_let_me_go',
+    'gone_with_the_wind',
+    'it',
+    'interpreter_of_maladies',
+    '1Q84',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const mainHeading = document.createElement(`h1`);
+  mainHeading.setAttribute(`class`, `mainHeading`);
+  mainHeading.innerText = `My Favorite Books`;
+  document.body.appendChild(mainHeading);
+
+  function FavoriteBook(title, language, author) {
+    this.title = title;
+    this.language = language;
+    this.author = author;
+  }
+
+  // eslint-disable-next-line no-new-object
+  const favoriteBooks = new Object();
```

Comment:
interesting choice to use `new Object()` instead of an object literal. Can you explain why you did that?